### PR TITLE
Migrate paper from notion to github

### PR DESCRIPTION
**Pull request by kerzhner**
As we prepare to publish the blog post, publicly available content will be on Github, not Notion.

### CODE REVIEW

File: doc/SAFE.md [hunk 1]
```diff
@@ -0,0 +1,405 @@
+# Secure Asymmetric Frugal Exchange (SAFE)
```
Comment (andrewgordstewart) [inline doc/SAFE.md:1]: I assume this is a copy/paste?

**Review by andrewgordstewart — COMMENTED** [review]
Do you think we should review the content now that it's a month old? Or merge it as is, and review it post-merge?

File: doc/SAFE.md (hunk 1 above)
Comment (kerzhner) [reply doc/SAFE.md:1]: Mostly copy/paste. The content is the same as in Notion, but some of the formatting had to be fixed in the `.md` file.

**Comment by kerzhner** [comment]
> Do you think we should review the content now that it's a month old? Or merge it as is, and review it post-merge?

My plan was:
1. Migrate the paper from Notion to git.
2. Lock the Notion doc.
3. Update + edit the paper via pull requests.

**Review by andrewgordstewart — APPROVED** [review]

**Pull request merged**